technical-reports/format/groups.md

```diff
@@ -95,11 +97,37 @@ Groups may support additional properties like type and description. Should other
 
 </div>
 
+### Type
+Groups may include an optional `type` property so a type property does not need to be manually added to every token. [See supported "Types"](#types) for more information.
+
```

Comment:
I think we should add something here to explain how the `type` property on a group should be interpreted. For example:

```suggestion

If a group has a `type` property it acts as a default type for any tokens within the group, including ones in nested groups, that do not explicity declare a type via their own `type` property. For the full set of rules by which a design token's type is determined, please refer to the [design token type property chapter](#type-0).

```

Reply:
I've updated the PR to include your suggestion @c1rrus.